grouper/fe/forms.py

```diff
@@ -264,3 +265,12 @@ class UserTokenForm(Form):
     name = StringField("Token name", [
         validators.DataRequired(),
     ])
+
+
+class UserPasswordForm(Form):
+    name = StringField("Password name", [
```

Comment:
Hmm. @rra should we make this user-defined, only have one, or constrain the set of possible password names?

(I assume this is intended to be used in the context of "service user is setting a password for")